### PR TITLE
OCM-8932 | ci: add support for imdsv2 for hcp cluster

### PR DESCRIPTION
**Pull request by aaraj7**
[aaraj@aaraj-thinkpadt14sgen1 rosa]$ rosa describe cluster -c 2cnqgcdb1r3fkhstogr7k9hc25e8g7mc -o json | jq -r .aws.ec2_metadata_http_tokens
required

### CODE REVIEW

File: tests/ci/data/profiles/rosa-hcp.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ profiles:
     byo_vpc: true
     private_link: false
     additional_principals: false
+    imdsv2: "required"
```
Comment (yasun1) [inline tests/ci/data/profiles/rosa-hcp.yaml:14]: is `imdsv2` achieved in the provisioning process now?

Comment (aaraj7) [reply tests/ci/data/profiles/rosa-hcp.yaml:14]: yes

**Pull request merged**